naga/tests/in/wgsl/conversions.wgsl

```diff
@@ -0,0 +1,99 @@
+// Conversion of initializer expressions
```

Comment:
I think some of this duplicates abstract-types-var.wgsl and abstract-types-const.wgsl? Would it make sense to merge into those and add an abstract-types-let.wgsl too?